src/main/java/econovation/moongtaengi/study/domain/Study.java

```diff
@@ -0,0 +1,35 @@
+package econovation.moongtaengi.study.domain;
+
+import econovation.moongtaengi.global.entity.BaseEntity;
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Embedded;
+import jakarta.persistence.Entity;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Table(name = "studies")
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class Study extends BaseEntity {
+    @Embedded
+    StudyName name;
+
+    @OneToMany(mappedBy = "study", cascade = CascadeType.ALL, orphanRemoval = true)
+    private List<StudyMember> members = new ArrayList<>();
+
+    public Study(StudyName name, Long hostId) {
+        this.name = name;
+        addMember(hostId, StudyRole.HOST); //스터디를 생성한 사람은 방장
+    }
+
+    public void addMember(Long memberId, StudyRole studyRole) {
+        StudyMember studyMember = new StudyMember(this, memberId, studyRole);
+        members.add(studyMember);
+    }
+}
```

Comment:
```suggestion
@Entity
@Table(name = "studies")
@Getter
@NoArgsConstructor(access = AccessLevel.PROTECTED)
public class Study extends BaseEntity {

    @Embedded
    private StudyName name;

    private Long hostId;  // 방장 ID만 저장

    public Study(StudyName name, Long hostId) {
        this.name = name;
        this.hostId = hostId;
    }
}
```
이런식으로 구성해서 관계를 최대한 사용하지 않는 방법은 어떤가요 ??

Reply:
인원이 만약 엄청 큰 오픈채팅방 같은 서비스를 생각한다면 Aggregate를 나누는 게 좋을 것 같긴 한데... 스터디라는 걸 생각했을 때 인원이 그렇게 많을 것 같지 않고 Study와 같은 Aggregate 내부에 두어서 인원 관리(만약 스터디 정원 제한이 있다면?), 방장 관리 등의 비즈니스 로직을 도메인 내부에서 관리하는 게 DDD 설계에 더 가까울 것 같아요!!